src/lib/components/Ideogram.react.js

```diff
@@ -477,15 +472,23 @@ Ideogram.propTypes = {
     dataDir: PropTypes.string,
 
     /**
-     * This is a work in progess and will hopefully be fixed in future releases.
+     * Organism(s) to show chromosomes for. Supply organism's name as a string (e.g. "human") or
```

Comment:
```suggestion
     * Organism(s) to show chromosomes for. Supply organism's name as a string (e.g., "human") or
```
Just because I'm going by the American style.

Reply:
I replaced it everywhere ;)

Reply:
It holds for *i.e.* as well, see https://github.com/plotly/dash-bio/pull/201#discussion_r263813029 ;)